tests/ok_comm/test_ok_comm_scripting.py

```diff
@@ -99,7 +99,7 @@ def side_effect(*args, **kwargs):
 
     expected_base_path = os.path.normcase(
         os.path.join(
-            os.path.dirname(get_current_file_abs_directory()),
+            os.path.dirname(os.path.dirname(get_current_file_abs_directory())),
```

Comment:
Since this file moved to a subfolder we now need to call dirname one more time to get back to the correct folder to match the path that will be used in ok_comm.py

Reply:
did we consider in the past using `os.path.join(get_current_file_abs_directory(), os.pardir, os.pardir)` to improve readability? I can't recall exactly.  It doesn't seem immediately obvious to me that a function called `dirname` would get the parent directory---but maybe that's just me